test/test-exec-deserialization.py

```diff
@@ -178,6 +178,34 @@ def test_removal(self):
 
         self.assertTrue(not os.path.exists(self.output_file))
 
+    def test_issue_6533(self):
+        content = '''
+        [Service]
+        ExecStart=/bin/sleep 5
+        '''
+
+        with open(self.unitfile_path, 'w') as f:
```

Comment:
I have misunderstood the logs in https://github.com/systemd/systemd/pull/6619#issuecomment-324735345, because the value of `self.unitfile_path` is `/run/systemd/system/test-issue-518.service` here. I think it would be better to name the generated service `/run/systemd/system/test-issue-6533.service` so that those who read the logs won't be confused.